### PR TITLE
Add mypy

### PR DESCRIPTION
**Pull request by kitagry**
SSIA

### CODE REVIEW

File: gokart/redis_lock.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 
 class RedisParams(NamedTuple):
-    redis_host: str
-    redis_port: str
-    redis_timeout: int
+    redis_host: Optional[str]
+    redis_port: Optional[int]
```
Comment (kitagry) [inline gokart/redis_lock.py:14]: port should be int.

File: gokart/slack/event_aggregator.py [hunk 2]
```diff
@@ -1,44 +1,49 @@
...
+    task: str
+    exception: str
+
+
 class EventAggregator(object):
 
     def __init__(self) -> None:
-        self._event_queue = defaultdict(list)
+        self._success_events: List[str] = []
+        self._failure_events: List[FailureEvent] = []
```
Comment (kitagry) [inline gokart/slack/event_aggregator.py:19]: Previous `_event_queue` has different types by the keys. It is difficult to specify the types. So, I separate the lists.

**Review by hirosassa — APPROVED** [review]
LGTM! Thanks!

**Review by Hi-king — APPROVED** [review]
cool! THX

**Comment by Hi-king** [comment]
@kitagry THX!

**Pull request merged**